src/org/zaproxy/zap/extension/ascanrulesBeta/InsecureHTTPMethod.java

```diff
@@ -451,5 +500,67 @@ private boolean testConnect (HttpMessage baseMsg, String thirdpartyHost, int thi
 		}
 		return false;
 	}
+	
+	private final MessageAndEvidence testHttpMethod(String httpMethod) throws IOException{
+		
+		httpMethod = httpMethod.toLowerCase().trim();
+		
+		if (httpMethod.equals("track")||httpMethod.equals("trace")||httpMethod.equals("connect")) {
+			/* We have separate methods for them */
+			return null;
+		}
+		
+        final HttpRequestHeader traceRequestHeader = this.getBaseMsg().getRequestHeader();
+        
+        traceRequestHeader.setMethod(httpMethod);
+        traceRequestHeader.setVersion(HttpRequestHeader.HTTP11);
+ 
+        final HttpMessage tracemsg = getNewMsg();
+        tracemsg.setRequestHeader(traceRequestHeader);
+        sendAndReceive(tracemsg, true); // do not follow redirects. That might ruin our day.
```

Comment:
Per the comment the second param should be `false`.
https://static.javadoc.io/org.zaproxy/zap/2.6.0/org/parosproxy/paros/core/scanner/AbstractPlugin.html#sendAndReceive-org.parosproxy.paros.network.HttpMessage-boolean-